packages/sdk/src/messageBackend/goldsky.ts

```diff
@@ -0,0 +1,433 @@
+import { BigNumber } from 'ethers';
```

Comment:
I noticed the functions in this file look very similar to the ones in sdk-bridge (packages/sdk-bridge/src/backend.ts), any way to consolidate here?

Reply:
Oh I see the input types are slightly different

Reply:
Addressed in the new commit